src/components/searchSection/myStyle.scss

```diff
@@ -0,0 +1,100 @@
+$gray-color: #979797;
+$pink-color: #eec197;
+$light-blue: #8bb4ea;
+$dark-blue: #7aa2d6;
+$light-gray: #f6f6f6;
+$light-blue2: #f9f9f9;
+$font-roboto: 'Roboto';
+$light-black: #0e0e0e;
+$light-pink: #fee4cb;
+.lightBlack {
+  background: $light-black;
+}
+
+.mycolorPink {
+  // background: $pink-color;
+  background: $light-pink;
+}
+.textPink {
+  color: $light-pink;
+}
+
+%fontType {
+  font-family: 'Saira';
+}
+
+.fontHeader1 {
+  @extend %fontType;
+  color: $dark-blue;
+}
```

Comment:
you are using the font in an inheritance way and your teammates using it as variable
so talk with them and only chose one way

for me I will prefer the variable method cuz it's only one property, if it's more than one property so inheritance will be the answer for sure